src/Components/RegistrationView/Steps/FifthStep.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import { PropTypes } from 'prop-types';
+import { faSyncAlt } from '@fortawesome/free-solid-svg-icons';
+import Button from '../../Button/Button';
+import submitRegistrationData from '../../../requests';
+
+function FifthStep({ formData, onSubmit }) {
+  const [backendConnectionError, setErrorState] = useState(false);
+  const [connectionMessage, setConnectionMessage] = useState('');
+  const messageClassName =
+    connectionMessage === 'Data submited. Redirecting to Login'
+      ? 'data-request-success-message'
+      : 'data-request-error-message';
+  const [counter, setCounter] = useState(5);
+
+  const sendRequestAndDisplayState = async () => {
+    submitRegistrationData(formData);
+    await Promise.resolve('Success')
+      .then(() => {
+        setConnectionMessage('Data submited. Redirecting to Login');
+        startCountdownAndRedirect();
+      })
+      .catch(() => {
+        setErrorState(true);
+        setConnectionMessage(
+          'Connection with backend failed. Data not saved. Please reload and try again!'
+        );
+      });
+  };
+
+  const startCountdownAndRedirect = () => {
+    setTimeout(onSubmit, 5000);
```

Comment:
What if someone somehow change view earlier? SetTimeout should be cleared just like intervals for the best practices